models/trigger.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const Joi = require('joi');
+const Regex = require('../config/regex');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .example(12345),
+
+    src: Joi
+        .string().regex(Regex.EXTERNAL_TRIGGER)
+        .example('~sd@1234:component'),
```

Comment:
why is `~ad@` syntax required here ? can't it be just `pipeline:jobName` format ?

Reply:
I think we're keeping `~sd@` for simplicity so we can reuse the `EXTERNAL_TRIGGER` regex to parse the `src` and `dest`.

Reply:
And I feel it's more consistent this way. Just in case we change the format we want for the EXTERNAL_TRIGGER later.

Reply:
Sounds good.